apps/web/src/app/[locale]/mypage/(github-custom)/OneType.tsx

```
@@ -1,73 +1,20 @@
 
-import { getGitanimalsLineString, GitanimalsLine } from '@/components/Gitanimals';
-import { useClientUser } from '@/utils/clientAuth';
-import { copyClipBoard } from '@/utils/copy';
-
-import { SelectPersonaList } from '../PersonaList';
-
-const DEFAULT_SIZE = { width: 600, height: 120 };
+import { LinePersonaSelect } from './LinePersonaSelect';
+import { LinePreview } from './LinePreview';
 
 interface Props {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)